routes/applications.ts

```diff
@@ -25,6 +25,5 @@ router.patch(
   applications.updateApplication
 );
 router.patch("/batch/update", authenticate, authorizeRoles([SUPERUSER]), applications.batchUpdateApplications);
```

Comment:
Can you also remove this and it's related code as this also performs the same thing since we are using /applications to add applications so by default we adding status and createdAt
and please check the flow once everything is working fine

Reply:
okay, just saw it was doing the same thing. Also, I was wondering, don't we have to remove tests of these APIs that we are removing?

Reply:
Yes you have to remove tests also